Firebase/Core/Public/FIRLoggerLevel.h

```diff
@@ -14,20 +14,22 @@
  * limitations under the License.
  */
 
+#import <GoogleUtilities/GULLoggerLevel.h>
```

Comment:
Now that GULLogger is nominally the public way to do this and FIRLogger is a compatibility shim for analytics, shouldn't we make FIRLoggerLevel a private header? I suppose that's a breaking change.